pages/test_analytics/importing_json.md.erb

```diff
@@ -49,6 +47,84 @@ For example, to import the contents of a `test-results.json` file in a Buildkite
 
 To learn more about passing through environment variables to `run_env`-prefixed fields, read [CI Environments](/docs/test-analytics/ci-environments).
 
+## How to import JSON in CircleCI
+
+To import [JSON-formatted test results](#json-test-results-data-reference), make a `POST` request to `https://analytics-api.buildkite.com/v1/uploads` with a `multipart/form-data` body, including as many of the following fields as possible in the request body:
+
+| Field name            | Value                                                             |
+|-----------------------|-------------------------------------------------------------------|
+| `data`                | [JSON-formatted test results](#json-test-results-data-reference)  |
+| `format`              | `json`                                                            |
+| `run_env[CI]`         | The name of your CI environment                                   |
+| `run_env[key]`        | A unique key for the build that initiated the CI run              |
+| `run_env[url]`        | A URL that links to the build in your CI system                   |
+| `run_env[branch]`     | A version control branch or reference for this run                |
+| `run_env[commit_sha]` | A unique identifier for the version control revision for this run |
+| `run_env[number]`     | A counter for this build                                          |
+| `run_env[message]`    | The commit message or other description of the build              |
+
+For example, to import the contents of a `test-results.json` file in a CircleCI pipeline run:
```

Comment:
Run reads like a stray verb on the 1st and 2nd go until you get the "oh!" moment. Is it possible to rephrase this a bit?